src/tests/data/FactoryUtils.ts

```diff
@@ -0,0 +1,13 @@
+class FactoryUtils {
+  public static create<T>(n: number, fn: () => T): T[] {
+    return Array(n).fill(null).map(fn);
+  }
+
+  public static mapToValue(arr, val) {
+    return arr.map((x) => x[val]);
+  }
+  public static compareByDate(a, b) {
+    return -1 * (new Date(a.date).getTime() - new Date(b.date).getTime());
+  }
+}
```

Comment:
Usually a good idea to document helper methods like these since they'll be used all throughout our tests so I'd suggest docstrings for each of these